### ISSUE TITLE
Selection border should not overlap the selection

### ISSUE THREAD

**Issue by Streetwalrus** [emersion/slurp#44]
Currently, slurp centers the border on the edge of the selection, rather than pushing it to the outside. This is especially noticeable when piping in predefined areas for selecting whole windows. This obscures the selected content when using thicker borders, and it's a bit confusing.
Both [slop](https://github.com/naelstrof/slop) and [hacksaw](https://github.com/neXromancers/hacksaw) draw the border around the selection with no overlap.

**Comment by emersion** [comment]
Yes. Here's the patch fixing this for mako: https://github.com/emersion/mako/commit/f8ddfe048072d85e1d78cb486a56634b1b93e200

**Comment by Streetwalrus** [comment]
There is one case I didn't consider when opening this issue: if you're trying to select an entire output and the border is outset from the selection, you won't be able to see it, so maybe this should be customizable.

**Comment by tmccombs** [comment]
I tried implementing this, and found another weird side-effect: the crosshairs are now offset from the border while dragging a selection. I'm not sure what the best way to handle that is (or for that matter, how slop or hacksaw handle it).

**Comment by KalaayPT** [comment]
are there any updates on this? been trying to figure out how to make my grim screenshots not have slurps black borders inside them
